### PR TITLE
AG-11993 Remove navigator min & max options

### PR DESCRIPTION
**Pull request by lsjroberts**
https://ag-grid.atlassian.net/browse/AG-11993

### CODE REVIEW

File: packages/ag-charts-community/src/chart/interaction/zoomManager.ts [hunk 1]
```diff
@@ -103,13 +102,20 @@ export class ZoomManager extends BaseManager<ZoomEvents['type'], ZoomEvents> imp
...
-    public guardMemento(blob: unknown): blob is ZoomMemento {
-        return (
-            isObject(blob) && (blob.ratioX != null || blob.ratioY != null || blob.rangeX != null || blob.rangeY != null)
-        );
+    public guardMemento(blob: unknown): blob is ZoomMemento | undefined {
+        if (blob == null) return true;
+        if (!isObject(blob)) return false;
+
+        for (const key of Object.keys(blob)) {
+            if (!['ratioX', 'ratioY', 'rangeX', 'rangeY'].includes(key)) {
```
Comment (alantreadway) [inline packages/ag-charts-community/src/chart/interaction/zoomManager.ts]: Nit: How type-safe is this array of keys - would it be better to define a `const` array of element type `keyof ZoomMemento` so we'll catch contract changes here?

**Review by alantreadway — COMMENTED** [review]
LGTM, but CI is failing

File: packages/ag-charts-community/src/chart/interaction/zoomManager.ts (hunk 1 above)
Comment (lsjroberts) [reply packages/ag-charts-community/src/chart/interaction/zoomManager.ts]: Good point, fixed.

**Review by alantreadway — APPROVED** [review]
LGTM

**Pull request merged**